noggin/controller/group.py

```diff
@@ -10,7 +10,7 @@
 from noggin.utility.controllers import group_or_404, with_ipa
 from noggin.utility.pagination import paginated_find
 from noggin.utility.templates import undo_button
-from noggin_messages import MemberSponsorV1
+from noggin_messages import MemberRemovedV1, MemberSponsorV1
```

Comment:
Because of this, you need a minimal version on `noggin-messages` in `pyproject.toml` (`^1.1.0`). Updating the lockfile is not enough.

Reply:
Ah, yes, totally missed that.